README.md

```diff
@@ -53,10 +53,10 @@ yarn test
 
 ### Run Storybook
 
-This project uses [Storybook][http://storybook.js.org/] for testing the look and behavior of its UI components.
+This project uses [Storybook](http://storybook.js.org/) for testing the look and behavior of its UI components.
 
 ```bash
-yarn storybook
+yarn start-storybook
```

Comment:
Can we omit these changes? While they're good changes and I would generally accept them, they're out of scope for this PR.

Reply:
Hi again - I filed a separate PR for this: https://github.com/adobe/parliament-ui-components/pull/204